go/client/chat_common.go

```diff
@@ -0,0 +1,27 @@
+// Copyright 2016 Keybase, Inc. All rights reserved. Use of
+// this source code is governed by the included BSD license.
+
+package client
+
+import (
+	"golang.org/x/net/context"
+
+	"github.com/keybase/client/go/libkb"
+	"github.com/keybase/client/go/protocol/keybase1"
+)
+
+func CheckUserOrTeamName(ctx context.Context, g *libkb.GlobalContext, name string) (*keybase1.UserOrTeamResult, error) {
```

Comment:
I was initially checking to see if the errs were not NotFound and short-circuiting, but that didn't work for TLF names so this is a workaround. Is there a way to see if a name is a valid TLF from client?